Build-Core.cmd

```diff
@@ -33,31 +38,31 @@ call %_dotnet% restore "%CMDHOME%\Build\Tools.csproj" --packages %TOOLS_PACKAGES
 
 SET CURRENT_CONFIGURATION=Debug
 
-call %_dotnet% restore /p:Configuration=%CURRENT_CONFIGURATION% "%SOLUTION%"
+call %_dotnet% restore /bl:%LOGFILENAME_PREFIX%-%CURRENT_CONFIGURATION%-Restore.binlog;ProjectImports=Embed /p:Configuration=%CURRENT_CONFIGURATION% "%SOLUTION%"
 @if ERRORLEVEL 1 GOTO :ErrorStop
 @echo RESTORE ok for %CURRENT_CONFIGURATION% %SOLUTION%
 
-call %_dotnet% build %BUILD_FLAGS% /p:Configuration=%CURRENT_CONFIGURATION% "%SOLUTION%"
+call %_dotnet% build %BUILD_FLAGS% /bl:%LOGFILENAME_PREFIX%-%CURRENT_CONFIGURATION%-Build.binlog;ProjectImports=Embed /p:Configuration=%CURRENT_CONFIGURATION% "%SOLUTION%"
```

Comment:
Keep in mind that this script won't work on MSBuild versions before 15.3. If that's OK with you then it's fine.

To preserve compat with all versions of MSBuild, use this instead:
https://www.nuget.org/packages/Microsoft.Build.Logging.StructuredLogger
`/logger:BinaryLogger,packages\Microsoft.Build.Logging.StructuredLogger.1.1.168\lib\net46\StructuredLogger.dll;%LOGFILENAME_PREFIX%-%CURRENT_CONFIGURATION%-Build.binlog`